README.md

```diff
@@ -241,6 +241,7 @@ Some [Groupware](#groupware) solutions also feature calendar/address book editin
   * [Ring](https://ring.cx/) - Free software for universal communication which respects freedoms and privacy of its users. ([Source Code](https://github.com/savoirfairelinux/ring-project)) `GPLv3` `C++`
   * [Rocket.Chat](https://rocket.chat/) - Teamchat solution similar to Gitter.im or Slack. ([Source Code](https://github.com/RocketChat/Rocket.Chat)) `MIT` `Nodejs`
   * [Scrollback](https://scrollback.io/) - Embeddable cross-platform chat for communities. ([Source Code](https://github.com/scrollback/scrollback)) `AGPLv3` `Nodejs`
+  * [Spectrum 2](http://spectrum.im/) - Spectrum 2 is an open source instant messaging transport.  It allows users to chat together even when they are using different IM networks. ([Source Code](https://github.com/hanzz/spectrum2)) `GPLv3` `C++`
```

Comment:
https://github.com/hanzz/spectrum2/search?utf8=%E2%9C%93&q=License&type= shows that most of it is under GPL, while some dependencies are under [Boost license](http://www.boost.org/LICENSE_1_0.txt) which is [MIT](https://en.wikipedia.org/wiki/MIT_License)-like (except you can redistribute in a compiled form without including the license). I'm fine with leaving only GPLv3.